common/plugins/eu.esdihumboldt.hale.common.instance/src/eu/esdihumboldt/hale/common/instance/model/InstanceUtil.java

```diff
@@ -252,6 +252,11 @@ else if (a instanceof Group) {
 			if (b instanceof Group && groupEqual((Group) a, (Group) b, propertyOrderRelevant))
 				return true;
 		}
+		else if (a instanceof BigDecimal && b instanceof BigDecimal) {
+			BigDecimal x = (BigDecimal) a;
+			BigDecimal y = (BigDecimal) b;
+			return x.compareTo(y) == 0;
```

Comment:
Would be nice if you could provide s short comment here why you do a special case handling for BigDecimal and how it makes a difference.